.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                python-version: ['3.8', '3.9', '3.10', '3.11']
+                python-version: ['3.9', '3.11', '3.12']
```

Comment:
We'll skip Python 3.10 in the docker stack so it is kind of pointless to test with it.